util/promiseAct.js

```diff
@@ -5,6 +5,7 @@ const Promise = require('bluebird');
  */
 module.exports = function promiseAct(runner, actName, fn) {
   return Promise.resolve(runner.actStart(actName))
+    .tap(() => runner.actEnd(actName))
     .then(fn)
-    .tap(() => runner.actEnd(actName));
-};
\ No newline at end of file
+    .catch(err => console.error(`Got an error in: ${actName}`, err));
```

Comment:
Agreed, I had a hard time debugging the script and added some `.catch(console.error)` calls myself, but I know it will swallow the exceptions from the runner's perspective.